src/Call.tsx

```diff
@@ -896,6 +894,9 @@ export class Call extends EventEmitter {
    * ```
    *
    * @param message The call message to send.
+   * Note that if `message.content` is of type `string`, it will be sent
+   * directly to Twilio as-is. If `message.content` is any other type, it will
+   * be processed using `JSON.stringify` before being sent to Twilio.
```

Comment:
Similar to my comment in the changelog, this is too verbose. Also, this describes CallMessage.content. Can we add this info here instead? https://github.com/twilio/twilio-voice-react-native/blob/latest/docs/api/voice-react-native-sdk.callmessage_interface.md